api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/data/types/RDFType.java

```diff
@@ -2,24 +2,35 @@
 
 import org.apache.jena.datatypes.RDFDatatype;
 import org.apache.jena.datatypes.xsd.XSDDatatype;
+import org.apache.jena.datatypes.xsd.impl.RDFLangString;
 
 import edu.cornell.mannlib.vitro.webapp.dynapi.components.Removable;
 import edu.cornell.mannlib.vitro.webapp.utils.configuration.Property;
 
 public class RDFType implements Removable {
 	
-	private static final String ANY_URI = "http://www.w3.org/2001/XMLSchema#anyURI";
+	public static final String ANY_URI = "http://www.w3.org/2001/XMLSchema#anyURI";
+	public static final String LANG_STRING = "langString";
```

Comment:
why it is not "http://www.w3.org/1999/02/22-rdf-syntax-ns#langString"? ANY_URI is defined as a complete data type url. 
Please, also note that LiteralParamFactory defines a const for http://www.w3.org/1999/02/22-rdf-syntax-ns#langString

Reply:
When we check literal we need full uri, when we create new RDFDataType, XSDDatatype takes only name as an argument. LiteralParamFactory is creating parameters in runtime, so for that it is not important if it is a name with namespace or not, but to define the same types in n3 it seemed to me more convenient to use short names the way they are required for  XSDDatatypes.